react/components/SummaryTotalizers/SummaryTotalizers.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FunctionComponent } from 'react'
+import SummaryItem from '../SummaryItem/SummaryItem'
+
+import { useSummary } from '../../Summary'
+
+const minTotalizerValue = 0
+const shippingData = {
+  id: 'Shipping',
+  name: 'Total do Frete',
+  value: minTotalizerValue,
+  __typename: 'Totalizer',
+}
+
+const isShippingPresent = (totalizers: Totalizer[]) => {
+  let result = false
+
+  for (let totalizer of totalizers) {
+    if (totalizer.id === 'Shipping') {
+      result = true
+    }
+  }
+
+  return result
+}
```

Comment:
It's not a change request just a tip, you can rewrite the whole function using `.find`:
`!!totalizers.find(t => t.id === 'Shipping')`

or `.some`:
`totalizers.some(t => t.id === 'Shipping')`

Reply:
done